templates/tools/dockerfile/clang_update.include

```diff
@@ -0,0 +1,32 @@
+#=================
+# Update clang to a version with improved tsan
+
+RUN apt-get update && apt-get -y install python cmake && apt-get clean
+
+RUN git clone -n -b release_38 http://llvm.org/git/llvm.git && ${'\\'}
```

Comment:
On Thu, Mar 17, 2016 at 7:29 AM, Vijay Pai <notifications@github.com> wrote:

> In templates/tools/dockerfile/clang_update.include
> <https://github.com/grpc/grpc/pull/5788#discussion_r56512494>:
>
> > @@ -0,0 +1,32 @@
> > +#=================
> > +# Update clang to a version with improved tsan
> > +
> > +RUN apt-get update && apt-get -y install python cmake && apt-get clean
> > +
> > +RUN git clone -n -b release_38 http://llvm.org/git/llvm.git && ${'\\'}
>
> What do you think of doing a checkout by a specific timestamp? That way we
> just have to change the desired timestamp each time we update.
>

I'd suggest going with this version of dockerfile for now to get rid of the
flake ASAP and figuring out the rest as a followup PR.

> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5788/files/b5b18faddc0f457623eef93bad362851cd3fcd19#r56512494>
>